### ISSUE TITLE
About tons of bugs 

### ISSUE THREAD

**Issue by Olebeh** [Androz2091/discord-player#1236]
Hello everyone who reads this. As you may know, in some of the "recent" updates (starting from November 2021, version 5.1.0) some of events have broken. Maybe that's not a problem, because there's already a DEV version that fixes them, but after downloading it even bigger disappointment already waits for us - nowPlaying event is not emitting, queue.current always returns undefined, so it causes command like "now playing" not to work and trying to play a Spotify song just crashes the bot. So looks like there's currently one fix - download an older version (e. g. 5.0.2 and earlier), but even now there're still problems - in this version we don't have a possibility to use custom stream engines, like play dl. So, for now the only thing community can do is wait. Wait for already really long time. Now, finally, questions everyone want to hear respond for: first, can we even expect for new stable version that will fix every bug I listed above and more (that I, maybe, haven't leasted), second, why this project is being so inactive, the last update was on March 6 not including discord.js v14 port update, aren't there any hot fixes for it, and the last one, is there any ETA of new version so we know how much should we wait more and do we even **have** to wait

I expect a decent and worhy reply from one of the creators just to hear respond to at least one of the questions. No negative, just want to use clear and almost bugless player, that is currently the best on the Internet. Everyone will wait for a reply. Thanks in advance

**Comment by lolollllo** [comment]
Sorry, im not a creator but I do have a fork of discord-player which I do publish new versions a lot. You can check it out at github.com/lolollllo/discord-portable-player

**Comment by lolollllo** [comment]
And also, would be glad if you could report all the bugs so I could get create a fix, Haven't noticed these issues since I don't use it.

**Comment by lolollllo** [comment]
Forked one.

**Comment by Androz2091** [comment]
@lolollllo, @Olebeh and to the other members of this community.

I'm still active and available to help maintaining this project. I'm willing to merge any corrective pull requests.

**Comment by larsrickert** [comment]
Hey @Olebeh,

I faced similar issues (especially waiting a long time for official release, not dev ones) and even wanted to contribute to this project. However, the code qualitity did not fullfill my personal standards, so I decided to create a new library from scratch written fully in strict TypeScript, called [discord-player-plus](https://discordplayerplus.lars-rickert.de/).

It uses play-dl, discord.js v14 and more. I don't want to poach users from this project but I think this community should know that there is an actively maintained alternative to `discord-player` on the market which publishes bug fixes etc. fast.

Competition is always useful for both sides so I hope that you are either satisfied with `discord-player-plus` or `discord-player` will start integrate fixes for the many know bugs into the main branch.
